README.md

```diff
@@ -1,97 +1,120 @@
-    |-----------------|------------------------------------------|
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply
-  - [ ] The Packages and Environments boxes can be unchecked
-- Search for any remaining TODOs and address them.
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Usage Example](#usage-example)
+  - [Return Example](#return-example)
+- [Contributing](#contributing)
+  - [Incrementing the Version](#incrementing-the-version)
+  - [Source Code Changes](#source-code-changes)
+  - [Recompiling Manually](#recompiling-manually)
+  - [Updating the README.md](#updating-the-readmemd)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+
+When the action runs it will add a deployment and deployment status record to the repo.  The GitHub Deployment will also have a special task d
 
 ## Inputs
 
-| Parameter | Is Required | Default | Description           |
-|-----------|-------------|---------|-----------------------|
-| `input`   | true        |         | Description goes here |
+| Parameter     | Is Required | Description                                                                                                                                                                                                                       |
+| ------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `token`       | true        | A token with `repo_deployment` permissions to create and update issues, workflows using this action should be granted `permissions` of `deployments: write`                                                                       |
+| `environment` | true        | The name of a GitHub environment the release was deployed to, i.e. [Dev\|QA\|Stage\|Demo\|UAT\|Prod]. It will be set in the standard `environment` property of the deployment object.                                             |
```

Comment:
Is this the right description? Isn't `environment` used for filtering when getting deployment?

Reply:
I'll try to make it more clear.